### ISSUE TITLE
Collect & display integrator logos in sign in flow

### ISSUE THREAD

**Issue by mkhandekar** [18F/myusa#733]
We have a new header in the sign in flow to communicate to visitors clearly that they are using MyUSA to sign into the integrator they are interacting with. To do this effectively, we need to be able to **collect**, and **display** the integrator logo in the header. 

If integrator **has a logo**, display it at same height as the MyUSA logo.

![2 2_signinemail](https://cloud.githubusercontent.com/assets/1598889/8944918/fdb222b4-3548-11e5-85e4-7729384e73bd.png)

If integrator **doesn’t have a logo**, display the app name in plain text in the same place.

![2 2_signintointegrator_nologo](https://cloud.githubusercontent.com/assets/1598889/8944919/fdb25720-3548-11e5-8ac1-a5f4d4da1058.png)

cc: @yozlet @harrisj @esgoodman

**Comment by harrisj** [comment]
Looks good. Do we want to enforce any constraints on how big the logo could be?

**Comment by mkhandekar** [comment]
@harrisj Yes! Integrator logo must be constrained to the same height as the MyUSA logo. I've updated the original issue too.
